src/state/update-user.js

```diff
@@ -1,23 +1,47 @@
-// import { useEffect, useState } from "react";
-
-export const updateUser = (uname, user) => {
-  console.log("there");
-  console.log("uname", uname, "user", user);
-  fetch(`https://m07.herokuapp.com/${uname}`, {
-    method: "POST",
-    headers: {
-      "Content-Type": "application/json"
-    },
-    mode: "cors",
-    body: JSON.stringify(user)
-  })
-    .then(response => response.json())
-    .then(() => {
-      console.log("now");
+import { useState, useEffect } from "react";
+const useUpdateUser = (initState, uname) => {
+  const [edit, setEdit] = useState(initState)
```

Comment:
Where is initState given a value?

Reply:
I passed the arguments when I called function